### DISCUSSION TITLE
[Docs] 组件代码示例建议换成 js

### DISCUSSION THREAD

**Discussion by elarbor** [element-plus/element-plus#15743] [Feature Request]
Feature type: **`Docs`**

## Details

- Enhanced component name: `All`

## Description

会 ts 的人肯定都会 js，而会 js 的人不一定会 ts。js 是原生代码，需要ts的人家一看 js 自己就知道怎么转换成 ts 的写法。但是看到ts代码，写js的人就不太会转换成js

### Additional comments

_(empty)_

**Comment by xiaowangxiaowang256256** [comment]
**问题描述：**
你指的是 这种《组件代码示例》么？
![image](https://github.com/element-plus/element-plus/assets/111274524/d311f4b3-8c4a-4617-ad84-364bdaf97163)

**解决方案：**
1、如果是只用js，直接将冒号后面的去掉就行
2、如果用的ts，不想写那么复杂可以，直接使用any替代

**Comment by kooriookami** [comment]
文档已更新
